src/tests/kdfs.rs

```diff
@@ -790,6 +793,7 @@ fn test_hkdf() {
 }
 
 #[cfg(feature = "pbkdf2")]
+#[cfg(not(feature = "no_sha1"))]
```

Comment:
This is eliminating a whole test when no_sha1 is selected, it'd be nice to add at least a vector with sha512 or so and disable only the sha1 ones.

Reply:
This is intentional:

https://github.com/latchset/kryoptic/blob/47bd87c101985df717d7e219798e58fb79fdb957/src/tests/kdfs.rs#L864 enforces SHA1

So to fix this issue, I will need some more test vectors with a different prf

Reply:
Would you mind to raise a separate issue for this? It is easy enough but it's a part from  different elephant

Reply:
I created https://github.com/latchset/kryoptic/issues/277